controllers/article.js

```diff
@@ -24,9 +24,16 @@ const {
 export default {
   async createArticle(req, res) {
     const {
-      title, body, description, category,
+      title, body, description, category, coverImageUrl
     } = req.body;
 
+    let imgUrl;
```

Comment:
What are you trying to do here? Why the change of variables?

Reply:
I had to give `coverImageUrl` a default value because of the `trim()` method when sending a post request. It fails because `Null` value cannot be trimmed.

Reply:
In that case, instead of this, on line 51, you could do:

`coverImageUrl: '' || coverImageUrl.trim()`

Reply:
Thanks, I've tried this but there's an error when I try to create an article without the `coverImageUrl`:
`Cannot read property 'trim' of undefined`

Reply:
`coverImageUrl: ''coverImageUrl && coverImageUrl.trim() || ''`

Reply:
Thanks, I've made the change.